src/metabase/api/task.clj

```diff
@@ -12,12 +12,14 @@
 
 (api.macros/defendpoint :get "/"
   "Fetch a list of recent tasks stored as Task History"
-  []
+  [_
+   {:keys [_status _task]
+    :as filter} :- task-history/Filter]
   (validation/check-has-application-permission :monitoring)
   {:total  (t2/count :model/TaskHistory)
    :limit  (request/limit)
    :offset (request/offset)
```

Comment:
:x: Pagination attributes should take `filter` into account. 

In the screenshot `total` should be `8`:

![image](https://github.com/user-attachments/assets/e0050c61-1ebd-4526-b67c-d7a31e691912)